src/PresentationalComponents/Cards/OverviewDashbarCard/components/DashbarCardTagOrIcon/DashbarCardTagOrIcon.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import propTypes from "prop-types";
+
+import { RouteIcon } from "@patternfly/react-icons";
+import { TagLabelWithTooltip } from "./TagLabelWithTooltip";
+import RuleLabels from "../../../../Labels/RuleLabels";
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+  INCIDENT_TAG,
+  IMPORTANT_TAG,
+  CRITICAL_TAG,
+  SEVERITY_MAP,
+} from "../../../../../AppConstants";
+
+export const DashbarCardTagOrIcon = ({ title }) => {
+  switch (title) {
+    case PATHWAYS:
+      return <RouteIcon size="md" data-cy="route-icon" />;
+    case INCIDENTS:
+      return (
+        <RuleLabels key="incidentTag" rule={{ tags: INCIDENT_TAG }} isCompact />
+      );
+    case CRITICAL_RECOMMENDATIONS:
+      return <TagLabelWithTooltip typeOfTag={SEVERITY_MAP[CRITICAL_TAG]} />;
+    case IMPORTANT_RECOMMENDATIONS:
+      return <TagLabelWithTooltip typeOfTag={SEVERITY_MAP[IMPORTANT_TAG]} />;
+
+    default:
+      return null;
```

Comment:
This could cause a React render error, or?

Reply:
I will change it to a React Fragment 
I think that would solve the issue you have mentioned 😇 
Thank you for noticing ! @bastilian